### PR TITLE
#357346 - Improve logs

### PR DESCRIPTION
**Pull request by caioqazz**
- Add SerilogAnalyzer nuget
- Copy Iris editorconfig and paste on this project
- Edit editorconfig to add [SerilogAnalyzer rules](https://github.com/Suchiman/SerilogAnalyzer)
- Fix logs

### CODE REVIEW

File: src/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+csharp_space_between_square_brackets = false
+
+# Xml project files
+[*.{csproj,vcxproj,vcxproj.filters,proj,nativeproj,locproj}]
+indent_size = 2
+
+# Xml config files
+
+# CA2000: Dispose objectes before losing scope
+dotnet_diagnostic.CA2000.severity = error
```
Comment (VictorBalbo) [inline src/.editorconfig:198]: Are this rules being applied? They are configured in the project files (`csproj,vcxproj,vcxproj.filters,proj,nativeproj,locproj`)  section but I think they should be on the C# section

File: src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs [hunk 2]
```diff
@@ -44,7 +44,7 @@ public override async Task ExecuteAsync(IContext context, ExecuteScriptSettings
             }
             catch (Exception e)
             {
-                _logger.Information(e.Message);
+                _logger.Information("{Message}", e.Message);
```
Comment (VictorBalbo) [inline src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs:47]: Maybe this error can be more usefull if we use a fixed message and pass the exception as the first parameter?
```suggestion
                _logger.Information(e, "Error converting timezone");
```

**Review by VictorBalbo — APPROVED** [review]

File: src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs (hunk 2 above)
Comment (caioqazz) [reply src/Take.Blip.Builder/Actions/ExecuteScript/ExecuteScriptAction.cs]: Fixed, thanks

File: src/.editorconfig (hunk 1 above)
Comment (caioqazz) [reply src/.editorconfig]: My bad, it was moved. Thanks!

**Review by tuliorezende — APPROVED** [review]

**Review by EmilsonTake — APPROVED** [review]

**Review by rafael-santiago-take — APPROVED** [review]

**Pull request merged**